### ISSUE TITLE
[BUG] lvm2 pv + encryption = seg fault, core dumped

### ISSUE THREAD

**Issue by markus-wa** [calamares/calamares#1847]
> Hi! Thank you for helping improve Calamares. If you are seeing a problem in installing a specific distribution, you should **probably** report the problem in the distribution's bug tracker, first. That helps filter out issues with packaging, mis-configuration, etc. that Calamares has no control over. If you are a distribution packager or maintainer, this page is for you.

**Describe the bug**
while trying to set up LVM2 with encryption on a 1TB ssd I keep getting seg fault, core dumped

**To Reproduce**
1. New Partition Table (GPT)
2. Create -> lvm2 pv -> encrypted -> set pw -> ok
3. Create Volume Group -> select pv -> enter name -> ok
4. Crash ...

**Expected behavior**
Be able to create volume group

**Screenshots and Logs**
If applicable, add screenshots to help explain your problem. Calamares has an installation log (usually `~/.cache/calamares/session.log`), please check it for confidential information and attach it if possible.

```
Dec 08 18:21:03 manjaro kernel: audit: type=1334 audit(1638987663.667:328): prog-id=100 op=UNLOAD
Dec 08 18:25:02 manjaro wpa_supplicant[1573]: wlp82s0: WPA: Group rekeying completed with e4:57:40:c3:1f:1c [GTK=CCMP]
Dec 08 18:26:51 manjaro audit[17569]: ANOM_ABEND auid=1000 uid=0 gid=0 ses=1 subj==unconfined pid=17569 comm="calamares" exe="/usr/bin/calamares" sig=11 res=1
Dec 08 18:26:51 manjaro kernel: calamares[17569]: segfault at 7f2e2c0a1 ip 00007f2e570eb20e sp 00007ffce85870d0 error 4 in libkpmcore.so.21.08.2[7f2e570ba000+9f000]
Dec 08 18:26:51 manjaro kernel: Code: 5e 48 89 e5 eb 1e 66 0f 1f 44 00 00 83 f8 ff 74 0a f0 83 2f 01 48 8b 3c 24 74 36 48 83 c3 08 49 39 dd 74 3b 48 8b 3b 48 8b 07 <ff> 90 98 00 00 00 48 89 c6 48 89 ef ff 15 d0 5a 0b 00 48 89 ee 4c
Dec 08 18:26:51 manjaro kernel: audit: type=1701 audit(1638988011.870:329): auid=1000 uid=0 gid=0 ses=1 subj==unconfined pid=17569 comm="calamares" exe="/usr/bin/calamares" sig=11 res=1
Dec 08 18:26:51 manjaro audit: BPF prog-id=103 op=LOAD
Dec 08 18:26:51 manjaro audit: BPF prog-id=104 op=LOAD
Dec 08 18:26:51 manjaro audit: BPF prog-id=105 op=LOAD
Dec 08 18:26:51 manjaro systemd[1]: Started Process Core Dump (PID 21274/UID 0).
░░ Subject: A start job for unit systemd-coredump@4-21274-0.service has finished successfully
░░ Defined-By: systemd
░░ Support: https://forum.manjaro.org/c/support
░░ 
░░ A start job for unit systemd-coredump@4-21274-0.service has finished successfully.
░░ 
░░ The job identifier is 4741.
Dec 08 18:26:51 manjaro audit[1]: SERVICE_START pid=1 uid=0 auid=4294967295 ses=4294967295 subj==unconfined msg='unit=systemd-coredump@4-21274-0 comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Dec 08 18:26:51 manjaro kernel: audit: type=1334 audit(1638988011.877:330): prog-id=103 op=LOAD
Dec 08 18:26:51 manjaro kernel: audit: type=1334 audit(1638988011.877:331): prog-id=104 op=LOAD
Dec 08 18:26:51 manjaro kernel: audit: type=1334 audit(1638988011.877:332): prog-id=105 op=LOAD
Dec 08 18:26:51 manjaro kernel: audit: type=1130 audit(1638988011.877:333): pid=1 uid=0 auid=4294967295 ses=4294967295 subj==unconfined msg='unit=systemd-coredump@4-21274-0 comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Dec 08 18:26:52 manjaro systemd-coredump[21275]: Process 17569 (calamares) of user 0 dumped core.
                                                 
                                                 Found module linux-vdso.so.1 with build-id: 21dd0b95b6d40459fda840e512d3ee4b5e26d389
                                                 Found module libxcb-res.so.0 with build-id: b825f66a75478fd00eeb4c57ecb0ac7792e00f5e
                                                 Found module KF5WindowSystemX11Plugin.so with build-id: 7aff4935a5964dc07d29efd673915749e8fe39ed
                                                 Found module libxml2.so.2 with build-id: b9441ffabb52b50c290152592f17cd78a381d493
                                                 Found module libncursesw.so.6 with build-id: 4118a7b0a2a0de8f6359df5b61c6c59a9e13a4d5
                                                 Found module libedit.so.0 with build-id: 4b0babfcad161c2ad0af6e59e2493258db23a331
                                                 Found module libffi.so.7 with build-id: de60e99f39569d11d09160bbdcd486cedc87d2b6
                                                 Found module libvulkan.so.1 with build-id: 07b49771f04f350af7587839fcec8b52fa4f3164
                                                 Found module libdrm_nouveau.so.2 with build-id: 90ddb582b57a61a46d1ebf6c993675f70dac7443
                                                 Found module libdrm_amdgpu.so.1 with build-id: a89ceb7c9082e5276f39023716eb4d194e75f6b8
                                                 Found module libelf.so.1 with build-id: cc912d9cef94d907317afba5edd9176e9828e776
                                                 Found module libdrm_radeon.so.1 with build-id: f6eef9ddb9d1db72080e709ce423f6bad7ea06f7
                                                 Found module libsensors.so.5 with build-id: dc8b2c1c0d8525411aca188ea3cc3fb86d381d30
                                                 Found module libLLVM-12.so with build-id: d1f4954a30a9c39d86e0b944607e0ca8020d651e
                                                 Found module nouveau_dri.so with build-id: 61aafa97fd156291d5b7c94a3bfb5b05abdd0d81
                                                 Found module libxshmfence.so.1 with build-id: 8876d9ccf620858795724ca24b9e567585a77cec
                                                 Found module libxcb-present.so.0 with build-id: 68f5465258750e2397b1333b3ffc01ee33caa4e1
                                                 Found module libxcb-dri3.so.0 with build-id: 9407a2480e09dc5a1dd9d9a0652fa8d32b328c91
                                                 Found module libXxf86vm.so.1 with build-id: 01e8243d0c6c971fa9e743e6e53d606b1cb76f6e
                                                 Found module libXext.so.6 with build-id: d70f24beb4fad748d6becffdcc13e51be0a2ebfa
                                                 Found module libxcb-dri2.so.0 with build-id: 2dd6e65129a809dab828a1d26215a3f7a363fcc8
                                                 Found module libdrm.so.2 with build-id: 3aeff5403ca8d7589eabc05752eb613937f454a1
                                                 Found module libglapi.so.0 with build-id: 57828fb70af7c1f2aa70b096d748851a1fd39a9c
                                                 Found module libGLX_mesa.so.0 with build-id: 71ea03043e73b18e4464477cb9dff4dfeeb19e95
                                                 Found module libcalamares_viewmodule_finished.so with build-id: 0ba8ecf295509b94eac25f962d87878994072feb
                                                 Found module libcalamares_job_initcpio.so with build-id: 15e72e38768434b3dc1f863bf02f7ca88c0e9f64
                                                 Found module libcalamares_job_luksbootkeyfile.so with build-id: ba2ec3505b21d17c7bfe375cd54781acd8fff47e
                                                 Found module libcalamares_job_machineid.so with build-id: 1257a23e0fa3549978c1917e9fa21aedd5204ffc
                                                 Found module libxcb-glx.so.0 with build-id: 8ad115382ad874165523dc5b4488c99d73bbc3c4
                                                 Found module libqxcb-glx-integration.so with build-id: c50ee6e2389cbc752234a3e0153821ca5c141f7c
                                                 Found module libcalamares_viewmodule_summary.so with build-id: cb2a3ba6b9bd75123af090275af86cedda5e499d
                                                 Found module libcrack.so.2 with build-id: 481466383400344261b84663137f13d6eb794554
                                                 Found module libpwquality.so.1 with build-id: a5d81bf15281b45fde13f30ff1a282ad00e565fc
                                                 Found module libcrypt.so.2 with build-id: 3743451bdaf36f951f926927633fd964813025d0
                                                 Found module libcalamares_viewmodule_users.so with build-id: 13016a1831b21e771047181a60cb555ac9b39880
                                                 Found module libpmsfdiskbackendplugin.so with build-id: 3c80998bf99c938001872bd9db6a10258e20ee1b
                                                 Found module libcalamares_viewmodule_partition.so with build-id: 9fc35d6aa764ef449d3bdfc3a72fcec701fc8977
                                                 Found module libcalamares_viewmodule_keyboard.so with build-id: cfc6f74b390dba3035e41d19e4687e7a916eaaf1
                                                 Found module libcalamares_viewmodule_locale.so with build-id: 42eb0f2e668cb1982f34a00919b24c851703944f
                                                 Found module libnss_dns.so.2 with build-id: b0d90fd1794c177c70f1e457c11ec1a7f87a41ed
                                                 Found module libnss_resolve.so.2 with build-id: 73f8ff308722aa502c4a598b911171351cad9fb5
                                                 Found module libnss_mdns4_minimal.so.2 with build-id: a6fde528b667185a18aca5be61727fb369423647
                                                 Found module libnss_mymachines.so.2 with build-id: 7d8490ec6c72f86b95cde5b4a5da4c13861bc0fc
                                                 Found module libqnmbearer.so with build-id: 15daf5dc2f7fa2d307de33e024a9fc8a10703423
                                                 Found module libqgenericbearer.so with build-id: 9729f696c8143bba233600621d6b8dd5c4a3876e
                                                 Found module libqconnmanbearer.so with build-id: 1b3841f6bb95441084391197a49052fc17cee3d4
                                                 Found module libudev.so.1 with build-id: aca8577d374861da6157b28fc1990dccf99ba5c1
                                                 Found module libdevmapper.so.1.02 with build-id: 743ce9368ef9d7add12c5355e4e1185ce8424f7e
                                                 Found module libparted.so.2 with build-id: 35e52421a1e2200a3efa77b4093bb48fc1a061b7
                                                 Found module libcalamares_viewmodule_welcome.so with build-id: a8c3844f01466e86b3c5abfcb0775c534658b04b
                                                 Found module libqsvgicon.so with build-id: 0715b2f7d8b7c628eeee54d77af83aef47675dc3
                                                 Found module libqsvg.so with build-id: a64dfbfd7c6cb62e4c72b3061bdca492811c066b
                                                 Found module libjpeg.so.8 with build-id: 5d102b1a7291a621bba049969d25ead0ccecc6cc
                                                 Found module libqjpeg.so with build-id: 650f4fbd34a38d2944c5129eca60199545ff51e5
                                                 Found module libqico.so with build-id: 64a41b997f0cad89c5f7079835cdb4fef89b8ef6
                                                 Found module libqgif.so with build-id: d93f9967ef07903e9a07599bd0a581d270320ce6
                                                 Found module libkvantum.so with build-id: ea33ff2a9a0cefae8911078b6b3f47d8108ec5aa
                                                 Found module libqt5ct-style.so with build-id: 315b522d65c585c6dbf36bc2ed4fd27b38a3a1ca
                                                 Found module libcomposeplatforminputcontextplugin.so with build-id: c62f6a764ff2c7500d0ba10ad7838cc122cdb62d
                                                 Found module libqt5ct.so with build-id: d66bf8834e12695e322c297d81a727b1e76217b6
                                                 Found module libnss_files.so.2 with build-id: 1a36dfc01d3a1010b2ee79766a24a8090a3266d5
                                                 Found module libXfixes.so.3 with build-id: 0a05c7e8714522bfbdd7c0027c3e2a94965664b0
                                                 Found module libXrender.so.1 with build-id: 97e0b9ab6ba96ebc86527cc2b3c3078aad8616b3
                                                 Found module libXcursor.so.1 with build-id: eb6315bcd7526aaf51b0a1307efdc587cea0a152
                                                 Found module libuuid.so.1 with build-id: 832a09e0d9568cc6dbe699472218ea2f79dc0f34
                                                 Found module libxcb-util.so.1 with build-id: 4188ebc629b472b560849c5792a69c1803c3d1a4
                                                 Found module libexpat.so.1 with build-id: 8850138eae6d9d4d43c5c4b2ac48393bc4279037
                                                 Found module libxkbcommon.so.0 with build-id: 1f1bc7527e57c886c3df5803068122e1971d4724
                                                 Found module libxkbcommon-x11.so.0 with build-id: 821e1e75e92f3aaa96d19ed31fe98dbbd4d34627
                                                 Found module libICE.so.6 with build-id: af53458a64fe1e62a4d95606f7b03c210b2b098c
                                                 Found module libSM.so.6 with build-id: ac7b055fd3f16b4f29373285ab04ed52eebb632d
                                                 Found module libxcb-xinput.so.0 with build-id: 6403174c48f05f40cf7c168beb86f2886b9f09c1
                                                 Found module libxcb-xkb.so.1 with build-id: fce20d2101f3e5439923e9283c489c2cad4420c3
                                                 Found module libxcb-xinerama.so.0 with build-id: b328103ce7502d94fd3a4e2e8773978a9aa84adc
                                                 Found module libxcb-xfixes.so.0 with build-id: a6b197ace5b9b59f913f5969eb419a88d1194f47
                                                 Found module libxcb-sync.so.1 with build-id: dda14591103b01b1311906053bf1ca9e82ade35c
                                                 Found module libxcb-shape.so.0 with build-id: f1b3376c4534fab0469feb1cd19f022ff98df533
                                                 Found module libxcb-render.so.0 with build-id: a37bdb37744b508be9dd29fd9ccb9bb0170d43e5
                                                 Found module libxcb-render-util.so.0 with build-id: c64fb28ea20a320a9754273d9ea3af85f6f74b17
                                                 Found module libxcb-randr.so.0 with build-id: b6c1c7d4b28863b55ade251813db7f246b7ffe8b
                                                 Found module libxcb-shm.so.0 with build-id: fb797f299a446f559a95afcc168227482cc800d1
                                                 Found module libxcb-image.so.0 with build-id: ee7f8577926fe7c1d6972036c2f8e6f727e43b1b
                                                 Found module libxcb-icccm.so.4 with build-id: 73f1e12d76af383aa969a60279ed5ed922045f47
                                                 Found module libX11-xcb.so.1 with build-id: 0db4f94d8ae31b8dc9a83f825a9171656f1e532c
                                                 Found module libfontconfig.so.1 with build-id: 1103a641395c7d3b42e49b793d3a9ea927c77bf6
                                                 Found module libQt5XcbQpa.so.5 with build-id: 1faa9fbbd961134122bbf8a60c0bf2b1e58bb216
                                                 Found module libqxcb.so with build-id: 840fc6ab5aef9669dfcbf5bd8a034f173cc8c0a2
                                                 Found module libbrotlicommon.so.1 with build-id: f68934f94312f770550ebc6c5acfd6359b098b07
                                                 Found module libgpg-error.so.0 with build-id: ba85170c2d9343ea05eea8fa2048c212ff4ef552
                                                 Found module libbrotlidec.so.1 with build-id: f871e6f204ab8d48099915126ba01be989a8000f
                                                 Found module libbz2.so.1.0 with build-id: 919597c477c9b2cb9cdbb7745ed6494ac0e6da60
                                                 Found module libXdmcp.so.6 with build-id: 8ca0792d23c8b8b4c0864297512349292bea5955
                                                 Found module libXau.so.6 with build-id: 1c67764663e07bec24d8951e5fd93f4d165979ff
                                                 Found module libresolv.so.2 with build-id: c915c72668282861a813f7ea3c0780f37b681dc0
                                                 Found module libkeyutils.so.1 with build-id: ac405ddd17be10ce538da3211415ee50c8f8df79
                                                 Found module libkrb5support.so.0 with build-id: adf65240a4d2aba772d7a0772b4d015469934113
                                                 Found module libcom_err.so.2 with build-id: eb61ef71c8b97846db759fb89a115405cff6dd30
                                                 Found module libk5crypto.so.3 with build-id: eb8220b8f36675aac769450be4cb6bb7f97ec38a
                                                 Found module libkrb5.so.3 with build-id: 72d26767c5cb1097db75a5f5bff88860233c902b
                                                 Found module libpcre.so.1 with build-id: 845483dd0acba86de9f0313102bebbaf3ce52767
                                                 Found module libicudata.so.69 with build-id: 0ab994a49ef1848499c4af333b3266f28432a922
                                                 Found module libgcrypt.so.20 with build-id: db45f5d5e0f7af1e77324fea1885f974619ad268
                                                 Found module libcap.so.2 with build-id: c1674f9082fedd415876b9f7d9712269163259b5
                                                 Found module liblz4.so.1 with build-id: e63600ab23b2f6997f42fac2fa56e1f02ce159a1
                                                 Found module liblzma.so.5 with build-id: 8b615460aa230708c5183f16bede67aa0437d95e
                                                 Found module libgraphite2.so.3 with build-id: 47761dc11e553f519cde97ed9ee985be12ccdae2
                                                 Found module libfreetype.so.6 with build-id: 65e7f4a1e5557b0ceeaa433e5356f857fe9c669b
                                                 Found module libGLX.so.0 with build-id: 0f40ceaa036edc65147e9559b94e0219472493fc
                                                 Found module libGLdispatch.so.0 with build-id: 195b34c1bb271f3b1162c897cfbcfb859e656b93
                                                 Found module libdbus-1.so.3 with build-id: e937e5ef5cc9606af5f5f5bae96be6314828d1de
                                                 Found module libKF5WidgetsAddons.so.5 with build-id: 6dd6940a69a4f47ff20b5b4c4d04e575aae140f5
                                                 Found module libKF5I18n.so.5 with build-id: f46b822ad5705a692eadf4a60668872925cb69fe
                                                 Found module libblkid.so.1 with build-id: f03a24e92c9852272bd0049b5bcb7ed6cf0e664f
                                                 Found module librt.so.1 with build-id: 75484da2d6f1515189eefa076e0a40328834cd16
                                                 Found module libutil.so.1 with build-id: af6d8200064ee6e6a01252884d166b44f0a6121b
                                                 Found module libxcb-keysyms.so.1 with build-id: 333e5eb4075795f947c924a2085f17e2e2ae7147
                                                 Found module libxcb.so.1 with build-id: 0d1ef11740a5daad2ee331e812a51aa6574af222
                                                 Found module libcrypto.so.1.1 with build-id: 6d23f0a3f354825868d044684fad31d482cc9210
                                                 Found module libssl.so.1.1 with build-id: 959cd8f1dd43fc31aeb93caac4eda87c907ef18b
                                                 Found module libgssapi_krb5.so.2 with build-id: e6e098ad51ce7bdd3dbe902d7b0f69a90f8a9e08
                                                 Found module ld-linux-x86-64.so.2 with build-id: 040cc3dd10461562f177df39e3be2f3704258c3c
                                                 Found module libglib-2.0.so.0 with build-id: 1fe1592d097fa28822c23e96d9f357583b48424d
                                                 Found module libzstd.so.1 with build-id: 4b10444c1560ebc574af4d5f488b7408b22d450e
                                                 Found module libpcre2-16.so.0 with build-id: 983354d71426f38a37695490499fc89d050d7cc5
                                                 Found module libdl.so.2 with build-id: 5abc547e7b0949f89f3c0e21ab0c8331a7440a8a
                                                 Found module libicuuc.so.69 with build-id: 5cf18c56e2f64efdac32cf61fb9c0c48e9bb1797
                                                 Found module libicui18n.so.69 with build-id: 9cdecde5b2e47a2bd81dc14915cbfefcade76c12
                                                 Found module libdouble-conversion.so.3 with build-id: 5bffb4317594514068e082d4b4e9acd99987ec47
                                                 Found module libsystemd.so.0 with build-id: 58c6a86edd7f38011bd682024a3b53b8a963bd30
                                                 Found module libmd4c.so.0 with build-id: 2151dba72cdc9960097e6e960a58a2f03c8bb96e
                                                 Found module libharfbuzz.so.0 with build-id: f954dfb80265c0dd2484e766282305a739b8239d
                                                 Found module libz.so.1 with build-id: 81bf6e728a6d6f5b105b0f8b25f6c614ce10452a
                                                 Found module libpng16.so.16 with build-id: 2dc0bce07f199bf983c07a05fb95a6f4af83a9b3
                                                 Found module libGL.so.1 with build-id: 8b3028bab286029945569a0dfcf695f0f61d9c3b
                                                 Found module libpthread.so.0 with build-id: 07c8f95b4f3251d08550217ad8a1f31066229996
                                                 Found module libQt5DBus.so.5 with build-id: eec55e409fa44577508e4ea9512ecaa9bd668b65
                                                 Found module libkpmcore.so.11 with build-id: 34351028b97e78d4ca0add145e4c902b8e9e66b9
                                                 Found module libboost_python39.so.1.76.0 with build-id: 36a4f7c30c6c59e77670a2588302252f11a8bb1e
                                                 Found module libpython3.9.so.1.0 with build-id: 44da06e8bfa38c81b0cbf272e4904b01563a4ec4
                                                 Found module libX11.so.6 with build-id: e9192497aa39fab51358826d1a1b5fb77edc5081
                                                 Found module libQt5X11Extras.so.5 with build-id: e41614d2e35852f913bf00633d635cfbf5a90dce
                                                 Found module libKF5WindowSystem.so.5 with build-id: 4465afe9665b61e245c8dcffff5587a9d11c535b
                                                 Found module libm.so.6 with build-id: 2b8fd1f869ecab4e0b55e92f2f151897f6818acf
                                                 Found module libQt5Network.so.5 with build-id: 764d42df9642f3caea52448b11932f570ba357d6
                                                 Found module libQt5QmlModels.so.5 with build-id: 6151af051434d5c08c002bbe0609df3e4d2e7d32
                                                 Found module libQt5Xml.so.5 with build-id: ec415d2e364d2b174ad5f9813ab2926098221a37
                                                 Found module libyaml-cpp.so.0.6 with build-id: 75b5e4762816d5c17af775519f9ba961ca7706aa
                                                 Found module libQt5Svg.so.5 with build-id: 926a697e0d991ee22ee414c13c9b75c253fc1d4d
                                                 Found module libc.so.6 with build-id: 4b406737057708c0e4c642345a703c47a61c73dc
                                                 Found module libgcc_s.so.1 with build-id: 7f8508bb914546ada778809b64b99d234337d835
                                                 Found module libstdc++.so.6 with build-id: 8ab0e57054dd1dcba681f217016afc6a4e639783
                                                 Found module libQt5Core.so.5 with build-id: 64315bf3c7e3c2dffa4cd71f908d5f98dc29d876
                                                 Found module libKF5CoreAddons.so.5 with build-id: e5541dcd552110922c3181fe43b60a337dab9758
                                                 Found module libQt5Qml.so.5 with build-id: 92006af8b04f29d7af10797e88969c614f450203
                                                 Found module libQt5Gui.so.5 with build-id: 0abf1d6cd108b3ee97d0f1d0bddb918bfd083352
                                                 Found module libQt5Quick.so.5 with build-id: 69525a381a38beecc12159340d46641bc8131c5c
                                                 Found module libQt5Widgets.so.5 with build-id: 7b1df52ee680499519fbfd11fd43ca8b3e2bc212
                                                 Found module libQt5QuickWidgets.so.5 with build-id: 40ef5bd18aebd12c92e079995c988e63d5225f4b
                                                 Found module libcalamares.so.3.2.44.3-1.3 with build-id: 10a3b868e683a7a4ff29dc640a2b4ebe414dc226
                                                 Found module libKF5Crash.so.5 with build-id: e971aff373671fdcdddc701a87d8abc24a64f6eb
                                                 Found module libcalamaresui.so.3.2.44.3-1.3 with build-id: 53e2c92faa99e00996debd7d62fefbc5dfaacd88
                                                 Found module calamares with build-id: 77013ab7da1b34075c216e3408d03bd0e0c04425
                                                 Stack trace of thread 17569:
                                                 #0  0x00007f2e570eb20e _ZN2FS7lvm2_pv6getPVsERK5QListIP6DeviceE (libkpmcore.so.11 + 0x7020e)
                                                 #1  0x00007f2e571058ad _ZN9LvmDevice13scanSystemLVMER5QListIP6DeviceE (libkpmcore.so.11 + 0x8a8ad)
                                                 #2  0x00007f2e401edebb _ZN19PartitionCoreModule13scanForLVMPVsEv (libcalamares_viewmodule_partition.so + 0x64ebb)
                                                 #3  0x00007f2e401ef17a _ZN19PartitionCoreModule23refreshAfterModelChangeEv (libcalamares_viewmodule_partition.so + 0x6617a)
                                                 #4  0x00007f2e40229d57 _ZN13PartitionPage23onNewVolumeGroupClickedEv (libcalamares_viewmodule_partition.so + 0xa0d57)
                                                 #5  0x00007f2e582aa8c5 n/a (libQt5Core.so.5 + 0x2fa8c5)
                                                 #6  0x00007f2e598ee817 _ZN15QAbstractButton7clickedEb (libQt5Widgets.so.5 + 0x254817)
                                                 #7  0x00007f2e598eeac0 n/a (libQt5Widgets.so.5 + 0x254ac0)
                                                 #8  0x00007f2e598f0758 n/a (libQt5Widgets.so.5 + 0x256758)
                                                 #9  0x00007f2e598f0989 _ZN15QAbstractButton17mouseReleaseEventEP11QMouseEvent (libQt5Widgets.so.5 + 0x256989)
                                                 #10 0x00007f2e59838fce _ZN7QWidget5eventEP6QEvent (libQt5Widgets.so.5 + 0x19efce)
                                                 #11 0x00007f2e597f4ff6 _ZN19QApplicationPrivate13notify_helperEP7QObjectP6QEvent (libQt5Widgets.so.5 + 0x15aff6)
                                                 #12 0x00007f2e597fce59 _ZN12QApplication6notifyEP7QObjectP6QEvent (libQt5Widgets.so.5 + 0x162e59)
                                                 #13 0x00007f2e582731ba _ZN16QCoreApplication15notifyInternal2EP7QObjectP6QEvent (libQt5Core.so.5 + 0x2c31ba)
                                                 #14 0x00007f2e597fb8ff _ZN19QApplicationPrivate14sendMouseEventEP7QWidgetP11QMouseEventS1_S1_PS1_R8QPointerIS0_Ebb (libQt5Widgets.so.5 + 0x1618ff)
                                                 #15 0x00007f2e59852e98 n/a (libQt5Widgets.so.5 + 0x1b8e98)
                                                 #16 0x00007f2e59856215 n/a (libQt5Widgets.so.5 + 0x1bc215)
                                                 #17 0x00007f2e597f4ff6 _ZN19QApplicationPrivate13notify_helperEP7QObjectP6QEvent (libQt5Widgets.so.5 + 0x15aff6)
                                                 #18 0x00007f2e582731ba _ZN16QCoreApplication15notifyInternal2EP7QObjectP6QEvent (libQt5Core.so.5 + 0x2c31ba)
                                                 #19 0x00007f2e58b86c50 _ZN22QGuiApplicationPrivate17processMouseEventEPN29QWindowSystemInterfacePrivate10MouseEventE (libQt5Gui.so.5 + 0x139c50)
                                                 #20 0x00007f2e58b5bb15 _ZN22QWindowSystemInterface22sendWindowSystemEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt5Gui.so.5 + 0x10eb15)
                                                 #21 0x00007f2e539b8800 n/a (libQt5XcbQpa.so.5 + 0x63800)
                                                 #22 0x00007f2e565784dc g_main_context_dispatch (libglib-2.0.so.0 + 0x544dc)
                                                 #23 0x00007f2e565cc749 n/a (libglib-2.0.so.0 + 0xa8749)
                                                 #24 0x00007f2e56575bc1 g_main_context_iteration (libglib-2.0.so.0 + 0x51bc1)
                                                 #25 0x00007f2e582ccada _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt5Core.so.5 + 0x31cada)
                                                 #26 0x00007f2e58271a6b _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt5Core.so.5 + 0x2c1a6b)
                                                 #27 0x00007f2e5827a258 _ZN16QCoreApplication4execEv (libQt5Core.so.5 + 0x2ca258)
                                                 #28 0x000055a8e7f9f032 main (calamares + 0x16032)
                                                 #29 0x00007f2e57bd8b25 __libc_start_main (libc.so.6 + 0x27b25)
                                                 #30 0x000055a8e7f9f9ee _start (calamares + 0x169ee)
                                                 
                                                 Stack trace of thread 21182:
                                                 #0  0x00007f2e57ca4b2f __poll (libc.so.6 + 0xf3b2f)
                                                 #1  0x00007f2e565cc6d9 n/a (libglib-2.0.so.0 + 0xa86d9)
                                                 #2  0x00007f2e56575bc1 g_main_context_iteration (libglib-2.0.so.0 + 0x51bc1)
                                                 #3  0x00007f2e582ccada _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt5Core.so.5 + 0x31cada)
                                                 #4  0x00007f2e58271a6b _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt5Core.so.5 + 0x2c1a6b)
                                                 #5  0x00007f2e5714b5f1 _ZN15ExternalCommand5startEi (libkpmcore.so.11 + 0xd05f1)
                                                 #6  0x00007f2e57150120 _Z9isMountedRK7QString (libkpmcore.so.11 + 0xd5120)
                                                 #7  0x00007f2e40173d85 n/a (libpmsfdiskbackendplugin.so + 0x8d85)
                                                 #8  0x00007f2e40174688 n/a (libpmsfdiskbackendplugin.so + 0x9688)
                                                 #9  0x00007f2e40175471 n/a (libpmsfdiskbackendplugin.so + 0xa471)
                                                 #10 0x00007f2e401773e6 n/a (libpmsfdiskbackendplugin.so + 0xc3e6)
                                                 #11 0x00007f2e401f13d7 _ZN19PartitionCoreModule12revertDeviceEP6Deviceb (libcalamares_viewmodule_partition.so + 0x683d7)
                                                 #12 0x00007f2e401f16ba _ZN19PartitionCoreModule16revertAllDevicesEv (libcalamares_viewmodule_partition.so + 0x686ba)
                                                 #13 0x00007f2e402112c2 n/a (libcalamares_viewmodule_partition.so + 0x882c2)
                                                 #14 0x00007f2e59e02b85 _ZThn16_N12QtConcurrent15RunFunctionTaskIvE3runEv (libcalamares.so.3.2.44.3-1.3 + 0x8fb85)
                                                 #15 0x00007f2e58082b42 n/a (libQt5Core.so.5 + 0xd2b42)
                                                 #16 0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #17 0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #18 0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 21173:
                                                 #0  0x00007f2e56fdd8ca __futex_abstimed_wait_common64 (libpthread.so.0 + 0x158ca)
                                                 #1  0x00007f2e56fd7574 pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf574)
                                                 #2  0x00007f2e58085964 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt5Core.so.5 + 0xd5964)
                                                 #3  0x00007f2e58082cf8 n/a (libQt5Core.so.5 + 0xd2cf8)
                                                 #4  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #5  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #6  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 17622:
                                                 #0  0x00007f2e56fdd8ca __futex_abstimed_wait_common64 (libpthread.so.0 + 0x158ca)
                                                 #1  0x00007f2e56fd7270 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf270)
                                                 #2  0x00007f2e026c197c n/a (nouveau_dri.so + 0x1c897c)
                                                 #3  0x00007f2e026bb058 n/a (nouveau_dri.so + 0x1c2058)
                                                 #4  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #5  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 17587:
                                                 #0  0x00007f2e57ca4b2f __poll (libc.so.6 + 0xf3b2f)
                                                 #1  0x00007f2e565cc6d9 n/a (libglib-2.0.so.0 + 0xa86d9)
                                                 #2  0x00007f2e56575bc1 g_main_context_iteration (libglib-2.0.so.0 + 0x51bc1)
                                                 #3  0x00007f2e582ccada _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt5Core.so.5 + 0x31cada)
                                                 #4  0x00007f2e58271a6b _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt5Core.so.5 + 0x2c1a6b)
                                                 #5  0x00007f2e5807e176 _ZN7QThread4execEv (libQt5Core.so.5 + 0xce176)
                                                 #6  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #7  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #8  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 17572:
                                                 #0  0x00007f2e57ca4b2f __poll (libc.so.6 + 0xf3b2f)
                                                 #1  0x00007f2e5613e63b n/a (libxcb.so.1 + 0xc63b)
                                                 #2  0x00007f2e5614037b xcb_wait_for_event (libxcb.so.1 + 0xe37b)
                                                 #3  0x00007f2e539b7768 n/a (libQt5XcbQpa.so.5 + 0x62768)
                                                 #4  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #5  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #6  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 17596:
                                                 #0  0x00007f2e57ca4b2f __poll (libc.so.6 + 0xf3b2f)
                                                 #1  0x00007f2e565cc6d9 n/a (libglib-2.0.so.0 + 0xa86d9)
                                                 #2  0x00007f2e56575bc1 g_main_context_iteration (libglib-2.0.so.0 + 0x51bc1)
                                                 #3  0x00007f2e582ccada _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt5Core.so.5 + 0x31cada)
                                                 #4  0x00007f2e58271a6b _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt5Core.so.5 + 0x2c1a6b)
                                                 #5  0x00007f2e5807e176 _ZN7QThread4execEv (libQt5Core.so.5 + 0xce176)
                                                 #6  0x00007f2e588f3509 n/a (libQt5Qml.so.5 + 0x329509)
                                                 #7  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #8  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #9  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 21172:
                                                 #0  0x00007f2e56fdd8ca __futex_abstimed_wait_common64 (libpthread.so.0 + 0x158ca)
                                                 #1  0x00007f2e56fd7574 pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf574)
                                                 #2  0x00007f2e58085964 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt5Core.so.5 + 0xd5964)
                                                 #3  0x00007f2e58082cf8 n/a (libQt5Core.so.5 + 0xd2cf8)
                                                 #4  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #5  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #6  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 21175:
                                                 #0  0x00007f2e56fdd8ca __futex_abstimed_wait_common64 (libpthread.so.0 + 0x158ca)
                                                 #1  0x00007f2e56fd7574 pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf574)
                                                 #2  0x00007f2e58085964 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt5Core.so.5 + 0xd5964)
                                                 #3  0x00007f2e58082cf8 n/a (libQt5Core.so.5 + 0xd2cf8)
                                                 #4  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #5  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #6  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 21177:
                                                 #0  0x00007f2e56fdd8ca __futex_abstimed_wait_common64 (libpthread.so.0 + 0x158ca)
                                                 #1  0x00007f2e56fd7574 pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf574)
                                                 #2  0x00007f2e58085964 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt5Core.so.5 + 0xd5964)
                                                 #3  0x00007f2e58082cf8 n/a (libQt5Core.so.5 + 0xd2cf8)
                                                 #4  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #5  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #6  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 17588:
                                                 #0  0x00007f2e57ca4b2f __poll (libc.so.6 + 0xf3b2f)
                                                 #1  0x00007f2e565cc6d9 n/a (libglib-2.0.so.0 + 0xa86d9)
                                                 #2  0x00007f2e56575bc1 g_main_context_iteration (libglib-2.0.so.0 + 0x51bc1)
                                                 #3  0x00007f2e582ccb57 _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt5Core.so.5 + 0x31cb57)
                                                 #4  0x00007f2e58271a6b _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt5Core.so.5 + 0x2c1a6b)
                                                 #5  0x00007f2e5807e176 _ZN7QThread4execEv (libQt5Core.so.5 + 0xce176)
                                                 #6  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #7  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #8  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 21171:
                                                 #0  0x00007f2e56fdd8ca __futex_abstimed_wait_common64 (libpthread.so.0 + 0x158ca)
                                                 #1  0x00007f2e56fd7574 pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf574)
                                                 #2  0x00007f2e58085964 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt5Core.so.5 + 0xd5964)
                                                 #3  0x00007f2e58082cf8 n/a (libQt5Core.so.5 + 0xd2cf8)
                                                 #4  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #5  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #6  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 21181:
                                                 #0  0x00007f2e56fdd8ca __futex_abstimed_wait_common64 (libpthread.so.0 + 0x158ca)
                                                 #1  0x00007f2e56fd7574 pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf574)
                                                 #2  0x00007f2e58085964 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt5Core.so.5 + 0xd5964)
                                                 #3  0x00007f2e58082cf8 n/a (libQt5Core.so.5 + 0xd2cf8)
                                                 #4  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #5  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #6  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 21174:
                                                 #0  0x00007f2e56fdd8ca __futex_abstimed_wait_common64 (libpthread.so.0 + 0x158ca)
                                                 #1  0x00007f2e56fd7574 pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf574)
                                                 #2  0x00007f2e58085964 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt5Core.so.5 + 0xd5964)
                                                 #3  0x00007f2e58082cf8 n/a (libQt5Core.so.5 + 0xd2cf8)
                                                 #4  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #5  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #6  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 21178:
                                                 #0  0x00007f2e56fdd8ca __futex_abstimed_wait_common64 (libpthread.so.0 + 0x158ca)
                                                 #1  0x00007f2e56fd7574 pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf574)
                                                 #2  0x00007f2e58085964 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt5Core.so.5 + 0xd5964)
                                                 #3  0x00007f2e58082cf8 n/a (libQt5Core.so.5 + 0xd2cf8)
                                                 #4  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #5  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #6  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 21180:
                                                 #0  0x00007f2e56fdd8ca __futex_abstimed_wait_common64 (libpthread.so.0 + 0x158ca)
                                                 #1  0x00007f2e56fd7574 pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf574)
                                                 #2  0x00007f2e58085964 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt5Core.so.5 + 0xd5964)
                                                 #3  0x00007f2e58082cf8 n/a (libQt5Core.so.5 + 0xd2cf8)
                                                 #4  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #5  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #6  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 17585:
                                                 #0  0x00007f2e57ca4b2f __poll (libc.so.6 + 0xf3b2f)
                                                 #1  0x00007f2e565cc6d9 n/a (libglib-2.0.so.0 + 0xa86d9)
                                                 #2  0x00007f2e56575bc1 g_main_context_iteration (libglib-2.0.so.0 + 0x51bc1)
                                                 #3  0x00007f2e582ccada _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt5Core.so.5 + 0x31cada)
                                                 #4  0x00007f2e58271a6b _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt5Core.so.5 + 0x2c1a6b)
                                                 #5  0x00007f2e5807e176 _ZN7QThread4execEv (libQt5Core.so.5 + 0xce176)
                                                 #6  0x00007f2e56ffeb0c n/a (libQt5DBus.so.5 + 0x15b0c)
                                                 #7  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #8  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #9  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 21179:
                                                 #0  0x00007f2e56fdd8ca __futex_abstimed_wait_common64 (libpthread.so.0 + 0x158ca)
                                                 #1  0x00007f2e56fd7574 pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf574)
                                                 #2  0x00007f2e58085964 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt5Core.so.5 + 0xd5964)
                                                 #3  0x00007f2e58082cf8 n/a (libQt5Core.so.5 + 0xd2cf8)
                                                 #4  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #5  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #6  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 17715:
                                                 #0  0x00007f2e57ca4b2f __poll (libc.so.6 + 0xf3b2f)
                                                 #1  0x00007f2e565cc6d9 n/a (libglib-2.0.so.0 + 0xa86d9)
                                                 #2  0x00007f2e56575bc1 g_main_context_iteration (libglib-2.0.so.0 + 0x51bc1)
                                                 #3  0x00007f2e582ccb57 _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt5Core.so.5 + 0x31cb57)
                                                 #4  0x00007f2e58271a6b _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt5Core.so.5 + 0x2c1a6b)
                                                 #5  0x00007f2e5807e176 _ZN7QThread4execEv (libQt5Core.so.5 + 0xce176)
                                                 #6  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #7  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #8  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
                                                 
                                                 Stack trace of thread 21176:
                                                 #0  0x00007f2e56fdd8ca __futex_abstimed_wait_common64 (libpthread.so.0 + 0x158ca)
                                                 #1  0x00007f2e56fd7574 pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf574)
                                                 #2  0x00007f2e58085964 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt5Core.so.5 + 0xd5964)
                                                 #3  0x00007f2e58082cf8 n/a (libQt5Core.so.5 + 0xd2cf8)
                                                 #4  0x00007f2e5807f3cb n/a (libQt5Core.so.5 + 0xcf3cb)
                                                 #5  0x00007f2e56fd1259 start_thread (libpthread.so.0 + 0x9259)
                                                 #6  0x00007f2e57caf5e3 __clone (libc.so.6 + 0xfe5e3)
░░ Subject: Process 17569 (calamares) dumped core
░░ Defined-By: systemd
░░ Support: https://forum.manjaro.org/c/support
░░ Documentation: man:core(5)
░░ 
░░ Process 17569 (calamares) crashed and dumped core.
░░ 
░░ This usually indicates a programming error in the crashing program and
░░ should be reported to its vendor as a bug.
Dec 08 18:26:52 manjaro systemd[1]: systemd-coredump@4-21274-0.service: Deactivated successfully.
░░ Subject: Unit succeeded
░░ Defined-By: systemd
░░ Support: https://forum.manjaro.org/c/support
░░ 
░░ The unit systemd-coredump@4-21274-0.service has successfully entered the 'dead' state.
Dec 08 18:26:52 manjaro audit[1]: SERVICE_STOP pid=1 uid=0 auid=4294967295 ses=4294967295 subj==unconfined msg='unit=systemd-coredump@4-21274-0 comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Dec 08 18:26:52 manjaro kernel: audit: type=1131 audit(1638988012.264:334): pid=1 uid=0 auid=4294967295 ses=4294967295 subj==unconfined msg='unit=systemd-coredump@4-21274-0 comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Dec 08 18:26:52 manjaro audit: BPF prog-id=105 op=UNLOAD
Dec 08 18:26:52 manjaro audit: BPF prog-id=104 op=UNLOAD
Dec 08 18:26:52 manjaro audit: BPF prog-id=103 op=UNLOAD
Dec 08 18:26:52 manjaro kernel: audit: type=1334 audit(1638988012.357:335): prog-id=105 op=UNLOAD
Dec 08 18:26:52 manjaro kernel: audit: type=1334 audit(1638988012.357:336): prog-id=104 op=UNLOAD
Dec 08 18:26:52 manjaro kernel: audit: type=1334 audit(1638988012.357:337): prog-id=103 op=UNLOAD
Dec 08 18:34:39 manjaro audit[21434]: USER_START pid=21434 uid=1000 auid=1000 ses=1 subj==unconfined msg='op=PAM:session_open grantors=pam_limits,pam_unix,pam_permit acct="root" exe="/usr/bin/pkexec" hostname=? addr=? terminal=? res=success'
```

**Additional context**
may be related to #1130

**Comment by adriaandegroot** [comment]
@highvoltage this looks like https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=992164 as well ?

**Comment by markus-wa** [comment]
I also have a more general question: is there currently any supported way to install a distro with LUKS + LVM with calamares?

**Comment by volitank** [comment]
I am having the same issue trying with the Debian Live Installer. I tried with Manjaro and also has the bug.

I cloned calamares main branch and still have this bug. Cannot create lvm + luks

**Comment by volitank** [comment]
I am not sure if this is related to LUKs directly. I seem to also be affected by this if I try to setup LVM manually without LUKS

**Comment by adriaandegroot** [comment]
> I also have a more general question: is there currently any supported way to install a distro with LUKS + LVM with calamares?

It might be possible by setting up the partitions first, then just setting mount points; I'd have to try it in a VM. The Debian bug has a slightly more readable backtrace, but it's easy enough to trigger also in a debug-enabled build on a live system.

**Comment by adriaandegroot** [comment]
Zero the tables, make a GPT table, start partitioning:
```
dd if=/dev/zero of=/dev/sda bs=1M count=64
parted /dev/sda mktable gpt
fdisk /dev/sda
```

The `fdisk` command is interactive:
- `n` for new, `1` for partition 1, enter, enter to accept the whole disk
- `t` to change type, `lvm`
- `w` to write changes and quit

Then carry on with LVM configuration:
```
pvcreate /dev/sda1
vgcreate myvg /dev/sda1
lvcreate -L5G -n myvol myvg
```

.. at this point I can get to manual partitioning, and configure up somethnig that kind of makes sense, but after that the installation fails anyway with being unable to create a partition on device `/dev/myvg`. Running the command that is shown in the error message .. gets me an error message that the command-line-arguments are wrong. So it might be something elsewhere **also** causing problems.

**Comment by geckolinux** [comment]
Possibly related crash: On a new 8GB VirtualBox disk I tried manually partitioning with a new partition table, LUKS encrypted XFS `/` partition and an _unencrypted_ Btrfs `/home` partition. (Yeah, I know, it's a dumb configuration). It also crashes reproducably on the LUKS creation step.

[log.txt](https://github.com/calamares/calamares/files/7700002/log.txt)

On a different install I was able to do an "Erase disk" with XFS + LUKS with no crashing.

Please let me know if you want me to create a new issue or not.

**Comment by adriaandegroot** [comment]
Starting over:
```
dd if=/dev/zero of=/dev/sda bs=1M count=64
parted /dev/sda mktable gpt
parted /dev/sda mkpart fat32 2048s 300M
parted /dev/sda mkpart lvm 301M '100%'
pvcreate /dev/sda2
vgcreate myvg /dev/sda2
lvcreate -L 5G -n rootvol myvg
lvcreate -L 5G -n homevol myvg
```

Make filesystems

```
mkfs.vfat /dev/sda1
mkfs.ext4 /dev/myvg/rootvol
mkfs.ext4 /dev/myvg/homevol
```

Start Calamares, note that the device is *myvg*, manual partition, edit the two partitions to give them a mountpoint. Do **not** check the *format* box; preserve the filesystem. Ignore the warning about the EFI partition -- because there's no EFI partition inside the PV, of course. Carry on with the installation .. oh, then it doesn't mount the parttiions, and proceeds to fill up `/tmp` instead until rsync fails.

@geckolinux the issue here is LVM, not LUKS -- please create a separate issue
@markus-wa I think the answer to your general question is "no, there does not seem to be any working LVM install procedure"

**Comment by volitank** [comment]
> @geckolinux the issue here is LVM, not LUKS -- please create a separate issue @markus-wa I think the answer to your 

@adriaandegroot Is it not possible for them to still be linked? Say it is an issue with /dev/mapper devices and not really LVM directly. That would make it effect both LUKS and LVM separately but still be the same issue.

**Comment by geckolinux** [comment]
Thanks for the reply @adriaandegroot . At any rate here's my separate bug report:
https://github.com/calamares/calamares/issues/1857

**Comment by JOJ0** [comment]
> I am not sure if this is related to LUKs directly. I seem to also be affected by this if I try to setup LVM manually without LUKS

Hi @volitank and @adriaandegroot, I can confirm that this happens without luks, just creating an lvm pv partition and then trying to create a volume group using it relates in: 
```
21:22:01 [2]: QXcbConnection: XCB error: 3 (BadWindow), sequence: 12309, resource id: 14750625, major code: 40 (TranslateCoords), minor code: 0
/usr/bin/calamares_polkit: line 3: 12450 Segmentation fault      sudo -E /usr/bin/calamares "$@"
```

@adriaandegroot when I installed Manjaro for the first time on a system about 2 months ago I worked around the installer issue by creating LVM stuff manually beforehand, similar to your descriptions. That works well yes :-)

**Comment by JOJ0** [comment]
Hi @adriaandegroot,

> Starting over:
> 
> ```
> dd if=/dev/zero of=/dev/sda bs=1M count=64
> parted /dev/sda mktable gpt
> parted /dev/sda mkpart fat32 2048s 300M
...
...
> Start Calamares, note that the device is _myvg_, manual partition, edit the two partitions to give them a mountpoint. Do **not** check the _format_ box; preserve the filesystem. Ignore the warning about the EFI partition -- because there's no EFI partition inside the PV, of course. Carry on with the installation .. oh, then it doesn't mount the parttiions, and proceeds to fill up `/tmp` instead until rsync fails.

I was trying to trick my installer again as I did some time ago once already, as pointed out earlier. I did it similar ot what you tried here. I now ran in exact the same issue of not mounting partitions and then filling up the wrong volume. It's odd, since I managed to install that same OS some time ago already by tricking the installer somehow. I just can't recall what else I did to make it work.

On thing that is missing in your description I _think_ is that the EFI partition should be mounted into /boot/efi, at least I think this is what Manjaro Linux assumes. Not sure about other distros. But still with that addition it doesn't make a difference, the volumes are not mounted and the rsync fails because it fills up the wrong disk ;-)

I will now try to mount the volumes manually before Calamares starts the rsync, let's see if that helps....,

**Comment by hrvoj3e** [comment]
@JOJ0 Did you try and did it work?  
Can you share details about *I will now try to mount the volumes manually before Calamares starts the rsync* ?
I am having the same problem and it drives me crazy.

**Comment by JOJ0** [comment]
@hrvoj3e IIRC it did not work and I gave up on Calamares entirely. My solution then was using manjaro-architect, a higly configurable alternative installer. Try that! It might cure your frustrations 😀 HTH

**Comment by hrvoj3e** [comment]
Finally. It works.

`calamares` has a bug with LVM - it unmounts and deactivates the whole LVM group before it can install OS at all.
Install goes on but cannot install bootloader because it works inside tmpfs the whole time.

The "fix" is to mount LVM volumes somewhere and `cd` into them before starting calamares.
This will stop volume groups to deactivate and installer will succeed. :partying_face: :tada: 

I reported this in an existing issue
 -  https://github.com/calamares/calamares/issues/1920#issuecomment-1101789100

**Comment by jkugler** [comment]
For anyone coming across this bug report, here is a here is how I got around it: https://joshuakugler.com/installing-kde-with-lvm-and-luks.html

**Comment by adriaandegroot** [comment]
There's a branch that tackles *some* of the issues -- a very old one, which I just rebased to current Calamares. I'm not going to put it in a release today, though.
